packages/zoe/test/unitTests/contracts/test-multipoolAutoswap.js

```diff
@@ -989,6 +992,188 @@ test('multipoolAutoSwap jig - swapOut', async t => {
+  };
+  const purses = [
+    moolaPurse,
+    moolaLiquidityIssuer.makeEmptyPurse(),
+    centralPurse,
+    simoleanPurse,
+    simoleanLiquidityIssuer.makeEmptyPurse(),
+  ];
+  const alice = await makeTrader(purses, zoe, publicFacet, centralR.issuer);
+
+  let mPoolState = {
+    c: 0,
+    s: 0,
+    l: 0,
+    k: 0,
+  };
+
+  // this test uses twice as much Central as Moola to make the price difference
+  // more visible.
+  const initMoolaLiquidityDetails = {
+    cAmount: centralTokens(10000),
+    sAmount: moola(5000),
+    lAmount: moolaLiquidity(10000),
+  };
+  const initMoolaLiquidityExpected = {
+    c: 10000,
+    s: 5000,
+    l: 10000,
+    k: 50000000,
+    payoutC: 0,
+    payoutS: 0,
+    payoutL: 10000,
+  };
+  await alice.initLiquidityAndCheck(
+    t,
+    mPoolState,
+    initMoolaLiquidityDetails,
+    initMoolaLiquidityExpected,
+    mIssuerKeywordRecord,
+  );
+  mPoolState = updatePoolState(mPoolState, initMoolaLiquidityExpected);
+
+  let sPoolState = {
+    c: 0,
+    s: 0,
+    l: 0,
+    k: 0,
+  };
+  const initSimoleanLiquidityDetails = {
+    cAmount: centralTokens(10000),
+    sAmount: simoleans(10000),
+    lAmount: simoleanLiquidity(10000),
+  };
+  const initSimLiqExpected = {
+    c: 10000,
+    s: 10000,
+    l: 10000,
+    k: 100000000,
+    payoutC: 0,
+    payoutS: 0,
+    payoutL: 10000,
+  };
+  const sIssuerKeywordRecord = {
+    Secondary: simoleanR.issuer,
+    Liquidity: simoleanLiquidityIssuer,
+  };
+
+  await alice.initLiquidityAndCheck(
+    t,
+    sPoolState,
+    initSimoleanLiquidityDetails,
+    initSimLiqExpected,
+    sIssuerKeywordRecord,
+  );
+  sPoolState = updatePoolState(sPoolState, initSimLiqExpected);
+
+  // trade for central specifying 300 output: moola price 155
+  // Notice that it takes half as much moola as the desired Central
+  const gain = 300;
+  const mPrice = priceFromTargetOutput(gain, mPoolState.c, mPoolState.s, 30);
+
+  const moolaIn = 160;
+  t.is(mPrice, 155);
+  const tradeDetailsB = {
+    inAmount: moola(moolaIn),
+    outAmount: centralTokens(gain),
+  };
+
+  const expectedB = {
+    c: mPoolState.c - gain,
+    s: mPoolState.s + mPrice,
+    l: 10000,
+    k: (mPoolState.c - gain) * (mPoolState.s + mPrice),
+    out: gain,
+    in: moolaIn - mPrice,
+  };
+  await alice.tradeAndCheck(
+    t,
+    false,
+    mPoolState,
+    tradeDetailsB,
+    expectedB,
+    mIssuerKeywordRecord,
+  );
+  mPoolState = updatePoolState(mPoolState, expectedB);
+
+  // trade for moola specifying 250 output: central price: 495, roughly double.
+  const gainC = 250;
+  const mPriceC = priceFromTargetOutput(gainC, mPoolState.s, mPoolState.c, 30);
+  t.is(mPriceC, 495);
+
+  const tradeDetailsC = {
+    inAmount: centralTokens(mPriceC),
+    outAmount: moola(gainC),
+  };
+
+  const expectedC = {
+    c: mPoolState.c + mPriceC,
+    s: mPoolState.s - gainC,
+    l: 10000,
+    k: (mPoolState.c + mPriceC) * (mPoolState.s - gainC),
+    out: gainC, // 250 should be 0??
```

Comment:
What was this comment meaning?

Reply:
It was a debugging note to myself.

I keep learning about things I need to do to make those more visible so I can find them up before I submit.